src/communication/CommandLine.cpp

```diff
@@ -0,0 +1,475 @@
+                            while (slice.scene.extruders.size() < extruder_count)
+                            {
+                                slice.scene.extruders.emplace_back(slice.scene.extruders.size(), &slice.scene.settings);
+                            }
+                        }
+                        break;
+                    case 'e':
+                    {
+                        size_t extruder_nr = stoul(argument.substr(1));
+                        while (slice.scene.extruders.size() <= extruder_nr) //Make sure we have enough extruders up to the extruder_nr that the user wanted.
+                        {
+                            slice.scene.extruders.emplace_back(extruder_nr, &slice.scene.settings);
+                        }
+                        last_settings = slice.scene.extruders[extruder_nr].settings;
+                        break;
+                    }
+                    case 'l':
+                    {
+                        argument_index++;
+                        if (argument_index >= arguments.size())
+                        {
+                            logError("Missing model file with -l argument.");
+                            exit(1);
+                        }
+                        argument = arguments[argument_index];
+
+                        const FMatrix3x3 transformation = last_settings.get<FMatrix3x3>("mesh_rotation_matrix"); //The transformation applied to the model when loaded.
+
+                        if (!loadMeshIntoMeshGroup(&slice.scene.mesh_groups[mesh_group_index], argument.c_str(), transformation, last_extruder.settings))
+                        {
+                            logError("Failed to load model: %s\n", argument.c_str());
+                            exit(1);
+                        }
+                        else
+                        {
+                            last_settings = slice.scene.mesh_groups[mesh_group_index].meshes.back().settings;
+                        }
+                        break;
+                    }
+                    case 'o':
+                        argument_index++;
+                        if (argument_index >= arguments.size())
+                        {
+                            logError("Missing output file with -o argument.");
+                            exit(1);
+                        }
+                        argument = arguments[argument_index];
+                        if (!FffProcessor::getInstance()->setTargetFile(argument.c_str()))
+                        {
+                            logError("Failed to open %s for output.\n", argument.c_str());
+                            exit(1);
+                        }
+                        break;
+                    case 'g':
+                        last_settings = slice.scene.mesh_groups[mesh_group_index].settings;
+                        /* ... falls through ... */
```

Comment:
We might as well break here - if that's easier to fix the compiler warning.

Reply:
Yeah, we could then go over it in the next iteration. But now it's also easier to follow the flow of the code I guess.